docs/notebooks/beta_regression.ipynb

```diff
@@ -0,0 +1,1557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    az.style.use("arviz-darkgrid")
</pre><p>This should be in a separated cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done</p>